### ISSUE TITLE
what version of keras,tensorflow and tensorflow_gpu is compatible to run this mask rcnn? 

### ISSUE THREAD

**Issue by Naveennavik** [matterport/Mask_RCNN#2346]
How to check the tensorflow versions compatibility with keras and run the code couldn t find any relevant information regarding this I'm using windows 10. and cuda 10.1 cudnn-7.6.5 @moorage @waleedka @PavlosMelissinos @haeric @rymalia

**Comment by olaviinha** [comment]
Not an answer to your question per se, but [hivie7510's fork](https://github.com/hivie7510/Mask_RCNN) has been updated to use the latest versions. Found it from this repo's [pull requests](https://github.com/matterport/Mask_RCNN/pull/2347). It appears to me this particular repository is perhaps no longer actively maintained.

**Comment by Ademord** [comment]
@olaviinha you saved me. all the other tf2 migrations do not work for me, omg thank you.
